cibuildwheel/__main__.py

```diff
@@ -67,6 +68,19 @@ def main() -> None:
                             run in Windows, and it will build and test for all versions of
                             Python. Default: auto.
                         ''')
+
+    parser.add_argument(
+        '--archs',
+        default=os.environ.get("CIBW_ARCHS", 'auto'),
+        help='''
+            Comma-separated list of CPU architectures to build for.
+            If unspecified, builds the architectures natively supported
+            on this machine. Set this option to build an architecture
+            via emulation, for example, using binfmt_misc and qemu.
+            Default: auto
+            Choices: auto, {}
```

Comment:
`auto` choice is already in the Architectures choices

Reply:
Ah, dang I think you reviewed while I was making a change! I've actually removed `auto` from the Architecture choices, it's now resolved into an Architecture at parse time.